testing/bdd/features/nesting/ebuttd_nested_elements.feature

```diff
@@ -97,7 +97,7 @@ Feature: Merging nested elements
         And the EBU-TT-Live document is valid
         And the EBU-TT-Live document is converted to EBU-TT-D
         Then EBUTTD document is valid
-        And span 2 has style "S2S6"
+        And span 3 has style "S2S6"
```

Comment:
I've forgotten how some of the spans megred but why does span 2 become 3 (test fails with 2 but not sure why)

Reply:
The new code is wrapping an intervening `<br/>` in a `<span>`, which means that the `<span>` whose style we were checking before is now at index 2 instead of index 1.